config/swagger.yaml

```diff
@@ -3042,6 +3055,31 @@ definitions:
       - log
       - return_value
       - return_type
+  GAObject:
+    type: object
+    properties:
+      caller_id:
+        $ref: '#/definitions/EncodedHash'
+      height:
+        type: integer
+      gas_price:
+        type: integer
+      gas_used:
+        type: integer
+      return_value:
+        $ref: '#/definitions/EncodedByteArray'
+      return_type:
+        description: The status of the call 'ok | error'.
+        type: string
+      inner_object:
+         $ref: '#/definitions/TxInfoObject'
```

Comment:
I see recursion TxInfoObject-GAObject

As I understand level of recursion is finite (2 levels?), maybe both the user and the tool could be further guided with more specific types re expected max nesting. Though complex too, and debatable, so not worth the extra effort. Ok.

Reply:
It's finite, but can be 100 levels deep if you want to pay for it. There is and should not be max nesting.